src/renderer/components/NewProjectBlock.tsx

```diff
@@ -1,34 +1,69 @@
-import { Box, colors, styled, Typography } from '@mui/material';
+import { Box, Paper, Stack, styled, Typography } from '@mui/material';
 import AddIcon from '@mui/icons-material/Add';
 import { useState } from 'react';
-import GenericSquareBox from './GenericSquareBox';
 import ModalContainer from './NewProjectModal/ModalContainer';
+import colors from '../colors';
 
-const NewProjectBox = styled(Box)`
-  background: ${colors.grey[900]};
-  width: calc(100vw - 80px);
+const NewProjectBox = styled(Paper)`
+  background: ${colors.grey[700]};
+  width: calc(100% - 40px);
```

Comment:
No need to change but for future reference. Because css property `box-sizing` is set to "border-box", can just add padding to parent element and set child to be width 100% instead of doing margin and then calculating width as 100% - margin in the child element

Reply:
True true